packages/application-generic/src/usecases/select-integration/select-integration.usecase.ts

```diff
@@ -52,6 +55,51 @@ export class SelectIntegration {
       return integrations[0];
     }
 
+    let integration: IntegrationEntity | null = null;
+
+    if (command.filterData.tenant) {
```

Comment:
if we have an integrationIdentifier in the overrides, I think it should go by  that one and not do it  by the tenant filters